src/components/atoms/Button/Button.tsx

```diff
@@ -6,6 +6,7 @@ import * as S from "./Button.styles";
 interface LinkProps {
   isLink?: boolean;
   linkTo?: string;
+  newTab?: boolean;
```

Comment:
the `newTab` changes can safely be be reverted in `Button` since it's superseded by `ButtonNG` in `RunTabToolbar`:
```typescript
<ButtonNG
  isLink
  newTab
// ...
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/f038fa7f07dd402677ba46b787847df727270978